### DISCUSSION TITLE
Support for ArtStation social media icon in profile

### DISCUSSION THREAD

**Discussion by SethFalco** [community/community#60461] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Could [ArtStation](https://www.artstation.com/) be added as an available icon for GitHub profiles?

I understand ArtStation is not a social media or software engineering platform. However, while GitHub may target engineers, it's used by a variety of disciplines. This makes sense as there are other teams involved in product development, and so can contribute on GitHub.

For example, artists:

* Comment on issues and attach designs (UI/UX)
* Submit PRs with new assets
* Review screenshots / static pages hosted after GitHub Actions

I've also been working with graphics and UI designers, and witnessed them participating in GitHub discussions to communicate with open-source maintainers and contribute to free and open-source software more.

It's not a substantial number of profiles that add it, compared to something like LinkedIn, but it'd be a good move imo, plus makes links look more attractive.

> About 707 results
> 
> — [`https://www.artstation.com/ "contributions in the last year" site:github.com`](https://www.google.com/search?q=https%3A%2F%2Fwww.artstation.com%2F+%22contributions+in+the+last+year%22+site%3Agithub.com&hl=en)

> About 377,000 results
> 
> — [`https://www.linkedin.com/in/ "contributions in the last year" site:github.com`](https://www.google.com/search?q=https%3A%2F%2Fwww.linkedin.com%2Fin%2F+%22contributions+in+the+last+year%22+site%3Agithub.com&hl=en)

This feature request is in a similar vein to the following:

* [Add support for Threads profiles](https://github.com/orgs/community/discussions/60271)
* [Stackoverflow profile](https://github.com/orgs/community/discussions/48265)

**Comment by thomson159** [comment]
Nice idea, I'm up!
